clients/hadoopfs/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## _Upcoming_
 
+## 0.2.2
+
+* Update lakeFS SDK to 1.12.0
+* Use overwrite in lakeFSLinker to control If-None-Match
```

Comment:
Sounds like a minor version bump?!  Or is this a bugfix that makes `overwrite` be fully atomic and correct (unlike in S3AFileSystem)?

Reply:
I guess that's q matter of perspective? I'd like to think about it as an "improvement" rather than q bug or a new feature